controllers/drpolicy_controller.go

```diff
@@ -382,6 +382,16 @@ func (r *DRPolicyReconciler) configMapMapFunc(configMap client.Object) []reconci
 		return []reconcile.Request{}
 	}
 
+	labelAdded := util.AddLabel(configMap, util.OCMBackupLabelKey, util.OCMBackupLabelValue)
+
+	if labelAdded {
+		if err := r.Update(context.TODO(), configMap); err != nil {
+			r.Log.Error(err, "Failed to add OCM backup label to ramen-hub-operator-config map")
+
+			return []reconcile.Request{}
+		}
+	}
```

Comment:
Why add label dynamically? Can we install a config map with a label when deploying?

Also when this code run? The commit message should describe the solution - why we add label dynamically on which events we add the label.

Reply:
@nirs yes, I also wanted to add the label only once upon initialization. But I didn't find the right place, so I consulted with @BenamarMk and he suggested to do it inside the update. Maybe I didn't understand him right, let's wait for his review.

Reply:
You absolutely can create it statically @ELENAGER

Reply:
I added the static label creatiion. Now ConfigMap will be initialized with OCM backup label.  Only in case, that user changed ConfigMap, we will ensure this label exists.